lib/language_pack/fetcher.rb

```diff
@@ -1,12 +1,23 @@
-require "yaml"
-require "language_pack/shell_helpers"
+require 'yaml'
+require 'language_pack/shell_helpers'
 
 module LanguagePack
   class Fetcher
     class FetchError < StandardError; end
 
     include ShellHelpers
-    CDN_YAML_FILE = File.expand_path("../../../config/cdn.yml", __FILE__)
+    CDN_YAML_FILE = File.expand_path('../../config/cdn.yml', __dir__)
+
+    OVERRIDE_VENDOR_URL = 'https://github.com/jimeh/' \
+                          'heroku-buildpack-ruby-binaries/' \
+                          'raw/master'.freeze
+    OVERRIDE_VENDOR_MAPPING = {
```

Comment:
Are these all the same?

Reply:
For now, yes. I'm actually planning to spend 20 minutes this morning to make it a bit more intelligent.